tokens/src/tests.rs

```diff
@@ -1773,6 +1773,94 @@ fn slashed_reserved_named_works() {
 		});
 }
 
+#[test]
```

Comment:
this is a very huge test file, should we split into multi test file under tests folder? @xlc

Reply:
#733 doesn't need to be a blocker for this